src/API/Notes.php

```diff
@@ -96,6 +96,18 @@ public function register_routes() {
 					'methods'             => \WP_REST_Server::DELETABLE,
 					'callback'            => array( $this, 'delete_all_items' ),
 					'permission_callback' => array( $this, 'update_items_permissions_check' ),
+					'args'                => array(
```

Comment:
It would be good to add unit test(s) around this new capability.

https://github.com/woocommerce/woocommerce-admin/blob/2c02a454a1dae31a27bb73d33171c429997f50b1/tests/api/admin-notes.php#L483

Reply:
@mattsherman good suggestion, I added a test case here: https://github.com/woocommerce/woocommerce-admin/pull/7743/commits/39a4647dafd468d9e01c6269eee998c786ac40c8